lockbox-ios/Extensions/UINavigationBar+.swift

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+extension UINavigationBar {
+    func addLockboxGradient() {
```

Comment:
Is using inheritance a better option here? Instead of each view controller having to call `addLockboxGradient()` the VC could just inherit from a LockboxViewController that handled the styling.